src/hooks/useFetch.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from "react";
+
+const useFetch = (url, options) => {
+  useEffect(() => {
```

Comment:
It's not a fully custom hook. It looks like an utility function. In the task you need to implement hook which allows you using it like
const {data, isLoading, state} = useFetchWithLogging(mealUrl) a
and use these values inthe component